server/cluster/cluster.go

```diff
@@ -62,7 +62,8 @@ import (
 
 var (
 	// DefaultMinResolvedTSPersistenceInterval is the default value of min resolved ts persistence interval.
-	DefaultMinResolvedTSPersistenceInterval = 10 * time.Second
+	// If interval in config is zero, it means not to persist resolved ts and check config with this DefaultMinResolvedTSPersistenceInterval
+	DefaultMinResolvedTSPersistenceInterval = time.Second
```

Comment:
Why need to reduce the interval?

Reply:
This default interval is also the interval for detecting whether the config has changed. During the manual test, it was found that the previous configuration was too long, and it took ten seconds for the configuration to take effect.